src/views/virtualmachines/utils/mappers.ts

```diff
@@ -14,3 +14,6 @@ export type VmimMapper = { [key: string]: { [key: string]: V1VirtualMachineInsta
 
 export const getVMIFromMapper = (vmiMapper: VmiMapper, vm: V1VirtualMachine) =>
   vmiMapper?.mapper?.[getNamespace(vm)]?.[getName(vm)];
+
+export const getVMIMFromMapper = (vmimMapper: VmimMapper, name: string, namespace: string) =>
```

Comment:
If it's not too much trouble, would you mind renaming `VmiMapper` and `VmimMapper` in this file to `VMIMapper` and `VMIMMapper` so they match our conventions?